templates/consent.html

```diff
@@ -0,0 +1,32 @@
+<amp-geo layout="nodisplay">
+  <script type="application/json">
+    {
+      "ISOCountryGroups": {
+        "eu": ["at", "be", "bg", "ch", "cy", "cz", "dk", "de", "ee", "el", "es", "fi", "fr", "hr", "ie", "is", "it", "li", "lv", "lt", "lu", "hu", "mt", "nl", "no", "pl", "pt", "ro", "si", "sk", "se", "uk"]
+      }
+    }
+  </script>
+</amp-geo>
+
+<amp-consent id="consent" layout="nodisplay">
+  <script type="application/json">
+    {
+      "consents": {
+        "eu": {
+          "promptIfUnknownForGeoGroup": "eu",
+          "promptUI": "consent-popup"
+        }
+      }
+    }
+  </script>
+
+  <div id="consent-popup" class="consent-popup">
+    <div>
+      <a href="#" on="tap:consent.dismiss" class="close-button" role="button" tabindex="0">✕</a>
```

Comment:
Can we move this to the right? Somehow I always expect close buttons to be on the right. Or is it just me?

Reply:
Its location varies from device to device!  I'm following the designs [here](https://docs.google.com/document/d/1j4zVW5oeyGwJ7Usg14JW4yXI4KrbY0FzNmgFxM2BZro/edit) and [here](https://gallery.googleplex.com/projects/MCHbtQVoQ2HCZeADcnNEYkpi/files/MCHtA7U1iMGr6yzAke-xKMPUgUfwZnEM3bg):

Reply:
sgtm